quickwit/quickwit-metastore/src/metastore/mod.rs

```diff
@@ -279,7 +279,7 @@ pub trait Metastore: Send + Sync + 'static {
 /// A query builder for listing splits within the metastore.
 pub struct ListSplitsQuery<'a> {
     /// The index to get splits from.
-    pub index: &'a str,
+    pub index_id: &'a str,
```

Comment:
This was annoying me a bit as well 😅 Although I'm not a huge fan of adding this small unrelated refactor in with the rest of the changes.

Reply:
Yes, I can split the PRs. This is already in an isolated commit.